### PR TITLE
a2 markup-a-readme

### PR DESCRIPTION
**Pull request by Juanmediina**
Creacion de la pagina A2

### CODE REVIEW

File: src/jm/html-css/a2-markup-a-readme/index.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (Iregui44) [inline src/jm/html-css/a2-markup-a-readme/index.html:5]:
```suggestion
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

File: src/jm/html-css/a2-markup-a-readme/index.html [hunk 2]
```diff
@@ -0,0 +1,131 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
```
Comment (Iregui44) [inline src/jm/html-css/a2-markup-a-readme/index.html:4]:
```suggestion
    <meta charset="UTF-8">
```

File: src/jm/html-css/a2-markup-a-readme/index.html [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  </head>
+  <body>
+    <h1>React</h1>
+    <p>React is JavaScript library for building user interfaces.</p>
+    <p></p>
+
+    <ul>
+      <li>
+        <strong>Declarative: </strong>react makes it painless to create
+        interactive Ul. Desing simple views for each states in your aplication,
```
Comment (Iregui44) [inline src/jm/html-css/a2-markup-a-readme/index.html:16]:
```suggestion
        interactive Ul. Desing simple views for each states in your application,
```

File: src/jm/html-css/a2-markup-a-readme/index.html [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+    <h3>Contributing Guide</h3>
+    <p>
+      Read our Contributing Guide to learn about out development process, we
+      have a list of good first issues that contain bugs that have a relatively
+      limited scope. This is a great place to get started
+    </p>
+
+    <h3>License</h3>
+    <p>React is MIT lisenced</p>
```
Comment (Iregui44) [inline src/jm/html-css/a2-markup-a-readme/index.html:129]:
```suggestion
    <p>React is MIT licensed</p>
```

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.multiRootWorkspaceName": "bootcamp-jul"
```
Comment (Iregui44) [inline .vscode/settings.json:2]: I think this settings should never be upload to git.

File: src/jm/html-css/a2-markup-a-readme/index.html [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+      </li>
+      <li>
+        <strong>Learn Once, Write Anywhere: </strong>We don't make assumptions
+        about the rest of your technology stack, so you can delevop new feature
+        in react woithout rewriting exiting code. React can also render on the
+        server using Node and power mobile apps using React Native
+      </li>
+    </ul>
+
+    <p>Learn how to us React in you project</p>
```
Comment (Iregui44) [inline src/jm/html-css/a2-markup-a-readme/index.html:36]: I think this must be a link. Using href

File: src/jm/html-css/a2-markup-a-readme/index.html (hunk 1 above)
Comment (Juanmediina) [reply src/jm/html-css/a2-markup-a-readme/index.html]: change made

File: src/jm/html-css/a2-markup-a-readme/index.html (hunk 2 above)
Comment (Juanmediina) [reply src/jm/html-css/a2-markup-a-readme/index.html]: change made

File: src/jm/html-css/a2-markup-a-readme/index.html (hunk 3 above)
Comment (Juanmediina) [reply src/jm/html-css/a2-markup-a-readme/index.html]: spelling corrected

File: src/jm/html-css/a2-markup-a-readme/index.html (hunk 4 above)
Comment (Juanmediina) [reply src/jm/html-css/a2-markup-a-readme/index.html]: spelling corrected

File: .vscode/settings.json (hunk 5 above)
Comment (Juanmediina) [reply .vscode/settings.json]: retired configuration

File: src/jm/html-css/a2-markup-a-readme/index.html (hunk 6 above)
Comment (Juanmediina) [reply src/jm/html-css/a2-markup-a-readme/index.html]: code changed to href

**Review by Iregui44 — APPROVED** [review]

**Pull request merged**